src/routes/api/configured/api-configured-post.ts

```diff
@@ -0,0 +1,45 @@
+import { Request, Response } from "express";
+import { Subscription } from "models/subscription";
+import { getLogger } from "config/logger";
+import { saveConfiguredAppProperties } from "utils/app-properties-utils";
+
+const MAX_INSTALLATIONS_BATCH = 50;
+/**
+ * This will update JIRA APP Properties for the GH4J installation to set isConfigured;
+ */
+export const ApiConfiguredPost = async (req: Request, res: Response): Promise<void> => {
```

Comment:
/nit
I think `AppConfigurationPost` or `AppConfigPost` sounds more appropriate than `ApiConfiguredPost`.

Reply:
yeah I bounced around a few i agree with `AppConfigurationPost` Ill update the rest to suit too